domains/omarmarquez.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Omar Márquez minimalist portfolio",
+    "repo": "https://github.com/Trkiller02/minimalist-portfolio",
+    "owner": {
+      "username": "Trkiller02",
+      "email": "trollkiller0207@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://trkiller02.github.io/minimalist-portfolio/"
```

what @omardotdev said
CNAME can't contain protcol or slashes
```suggestion
      "CNAME": "trkiller02.github.io"
```

---

> It turns out that my portfolio **is not located** on `"trkiller02.github.io"` but on `"https://trkiller02.github.io/minimalist-portfolio/"`

you still have to set it to `trkiller02.github.io` or else i think github pages wont work when trying to use your new subdomain..

---

> It turns out that my portfolio **is not located** on `"trkiller02.github.io"` but on `"https://trkiller02.github.io/minimalist-portfolio/"`

your CNAME would be trkiller02.github.io but you need to add custom domain on the website repo